docs/maya/muscle.md

```diff
@@ -184,6 +185,7 @@ To enable the debugger the *Debug* checkbox must be marked. To select the specif
 
  - **Attachments To Geometry**: For each vertex with a geometry attachment constraint weight greater than 0.0, a line will be drawn from the mesh vertex to its respective geometry target closest point at rest.
  - **Attachments To Transform**: For each vertex with a transform attachment constraint weight greater than 0.0, a line will be drawn from the mesh vertex to its respective transform target.
+  - **Fiber Constraints**: For each pair of vertices forming a constraint a line will be drawn. If the "Triangulate Mesh" option is disabled the debugged lines will align with the edges of the mesh polygons. If the "Triangulate Mesh" option is enabled the debugged lines will align with the edges of the underlying triangulation of the mesh.
```

Comment:
I think we agreed to refer to attributes with italic style instead of double quotes. If that is the case please apply this change wherever necessary.

Reply:
Done!